odbc/src/api.rs

```diff
@@ -1115,12 +1144,55 @@ pub extern "C" fn SQLSetEnvAttr(
 
 #[no_mangle]
 pub extern "C" fn SQLSetEnvAttrW(
-    _environment_handle: HEnv,
-    _attribute: EnvironmentAttribute,
-    _value: Pointer,
+    environment_handle: HEnv,
+    attribute: EnvironmentAttribute,
+    value: Pointer,
     _string_length: Integer,
 ) -> SqlReturn {
-    unimplemented!()
+    let env_handle = MongoHandleRef::from(environment_handle);
+    env_handle.clear_diagnostics();
+    match env_handle.as_env() {
+        None => SqlReturn::INVALID_HANDLE,
+        Some(env) => match attribute {
+            EnvironmentAttribute::OdbcVersion => match value as Integer {
+                380 => {
+                    let mut env_contents = (*env).write().unwrap();
+                    env_contents.attributes.odbc_ver = value as _;
+                    SqlReturn::SUCCESS
+                }
+                _ => {
+                    env_handle.add_diag_info(ODBCError::InvalidAttrValue("SQL_ATTR_ODBC_VERSION"));
+                    SqlReturn::ERROR
+                }
+            },
+            EnvironmentAttribute::ConnectionPooling => match value as Integer {
```

Comment:
umh, supposedly "SQL_ATTR_CONNECTION_POOLING is implemented inside the Driver Manager.", so we could just ignore whatever value is passer maybe?
PowerBi will set this to "SQL_CP_ONE_PER_HENV" so better not fails there or we won't be able to do anything :D
I don't think that the call will make to the driver, so it might be OK. We'll find out when we have a simple connection implemented.

Reply:
Oh interesting, maybe I can change it so that we only accept `SQL_CP_ONE_PER_HENV` instead of `SQL_CP_OFF`

Reply:
Filed [SQL-681 ](https://jira.mongodb.org/browse/SQL-681)so we can investigate this further